types/twilio-video/index.d.ts

```diff
@@ -279,13 +280,16 @@ export class RemoteTrackPublication extends TrackPublication {
     isTrackEnabled: boolean;
     kind: Track.Kind;
     track: RemoteTrack | null;
+    priority: TrackPriority;
```

Comment:
Isn't it `publishPriority`? https://media.twiliocdn.com/sdk/js/video/releases/2.3.0/docs/RemoteTrackPublication.html